pkgs/development/python-modules/python-forecastio/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage, stdenv, fetchPypi
+, requests
+, nose
+, coverage
+, responses
+}:
+
+buildPythonPackage rec {
+  pname = "python-forecastio";
+  version = "1.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0m6lf4a46pnwm5xg9dnmwslwzrpnj6d9agw570grciivbvb1ji0l";
+
+  };
+
+  checkInputs = [ nose coverage responses ];
+
+  propagatedBuildInputs = [ requests responses ];
+
+  checkPhase = ''
+    nosetests --with-coverage
```

Comment:
You should be able to just drop `--with-coverage` and the `coverage` dependency above.